### DISCUSSION TITLE
Export Page

### DISCUSSION THREAD

**Discussion by khanhleemtp** [prevwong/craft.js#411] [General]
How to export Editor as html,css,js when i want to build static file. And how to preview page in mobile, pc, tablet

**Comment by martin-dimi** [comment]
Did you find a solution @khanhleemtp I'm trying to do the same thing and am struggling.

**Comment by martin-dimi** [comment]
bump @prevwong

**Comment by ankri** [comment]
For your first question see: https://github.com/prevwong/craft.js/issues/447#issuecomment-1230523315

> And how to preview page in mobile, pc, tablet

Just open the page on mobile / PC / tablet?! Or: in devtools and use device emulation or use [sizzy](https://sizzy.app/)
